### PR TITLE
Fix various typos in classref

### PR DESCRIPTION
**Pull request by timothyqiu**
Found when translating the classref.

### CODE REVIEW

**Comment by MewPurPur** [comment]
Might as well add one I wrote down. Color.html_is_valid() GDScript example is missing a quotation mark.

File: doc/classes/OptionButton.xml [hunk 1]
```diff
@@ -6,9 +6,7 @@
 	<description>
 		[OptionButton] is a type of button that brings up a dropdown with selectable items when pressed. The item selected becomes the "current" item and is displayed as the button text.
 		See also [BaseButton] which contains common properties and methods associated with this node.
-		[b]Note:[/b] Properties [member Button.text] and [member Button.icon] are automatically set based on the selected item. They shouldn't be changed manually.
 		[b]Note:[/b] The ID values used for items are limited to 32 bits, not full 64 bits of [int]. This has a range of [code]-2^32[/code] to [code]2^32 - 1[/code], i.e. [code]-2147483648[/code] to [code]2147483647[/code].
-		[b]Note:[/b] The ID values used for items are 32-bit, unlike [int] which is always 64-bit. They go from [code]-2147483648[/code] to [code]2147483647[/code].
```
Comment (Sauermann) [inline doc/classes/OptionButton.xml:11]: Not sure, if this removal counts as a "typo".

**Review by Sauermann — APPROVED** [review]
The changes are looking good.

File: doc/classes/OptionButton.xml (hunk 1 above)
Comment (timothyqiu) [reply doc/classes/OptionButton.xml:11]: Line 9 and 11 are basically the same as Line 10 and 12.

https://github.com/godotengine/godot/blob/6758a7f8c07d1f4c8ec4f052ded6d26402967ebe/doc/classes/OptionButton.xml#L9-L12

**Review by YuriSizov — APPROVED** [review]

**Pull request merged**